mvc/src/main/java/webmvc/org/springframework/web/servlet/mvc/tobe/HandlerExecution.java

```diff
@@ -2,11 +2,41 @@
 
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
 import webmvc.org.springframework.web.servlet.ModelAndView;
 
 public class HandlerExecution {
 
+    private final Object handler;
+    private final Method handlerMethod;
+
+    public HandlerExecution(final Object handler,
```

Comment:
```suggestion
  protected HandlerExecution(final Object handler,
```
만약에 해당 패키지에만 생성되게 제한하실 생각이 있으시면 접근제어자도 변경하면 더 안전할것 같아요!!😊

Reply:
상속 구조를 사용할 것 같지는 않아서, 접근 제어자를 default 로 두었습니다! 감사합니다.